pkg/apis/addtoscheme_integreatly_v1alpha1.go

```diff
@@ -2,6 +2,7 @@ package apis
 
 import (
 	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
```

Comment:
can we not just import these from the keycloak-operator?

Like:
```
import kcv1alpha1 "github.com/integr8ly/keycloak-operator/pkg/apis/v1alpha1"
```

Rather than duplicating them in both repos.

Reply:
There is an issue with incompatible operator sdks between keycloak operator and the integreatly operator.

I have opened an issue for this -> https://github.com/integr8ly/integreatly-operator/issues/19